Formula/amazon-emr-on-eks-custom-image-cli.rb

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+require_relative '../ConfigProvider/config_provider'
+
+class AmazonEmrOnEksCustomImageCli < Formula
```

Comment:
This depends on python and docker, correct?  I'm not sure about adding docker as a dependency since I'm not sure how many people would be installed docker with homebrew, but what about python?

Reply:
Maybe a `caveat` is the best option forward: https://github.com/aws/homebrew-tap/blob/f89532a77230a2914d7e6e0d65041cfd76ad46da/Formula/eks-anywhere.rb#L18-L25 

You can just explain that docker is a dependency

Reply:
That sounds good! I updated the formula with caveats function, you could review and check that. Thanks!